packages/cicero-core/src/templatelibrary.js

```diff
@@ -127,6 +134,12 @@ class TemplateLibrary {
             return Promise.resolve(result);
         }
 
+        if(this.httpHeader) {
+            Logger.info('TemplateIndex with Authentication');
+        } else {
+            Logger.info('Template Index wwithout Authentication and Authorization');
```

Comment:
Typo here in `wwithout`. Also can you make the log messages symmetric between `getTemplate` and `getTemplateIndex`, please?